README.md

```diff
@@ -1,55 +1,139 @@
 
-Get the latest release [here](https://github.com/Autodesk/synthesis/releases/latest/).
-You can find all of our other releases on the [releases page](https://github.com/Autodesk/synthesis/releases/)
+Also note that the linux installer is tailored for Debian based distributions. Synthesis can be installed and ran on other distros but the installer is not guaranteed for your operating system.
 
-### Windows
+### Dependencies
 
-Download and run SynthesisWin[Version Number].exe.
+Synthesis Version `6.0.0` has the following dependencies:
 
-### MacOS
+- [.NET Standard 2.0](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0)
+- [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
+- [Protobuf 23.3](https://github.com/protocolbuffers/protobuf/releases/tag/v23.3)
+- Unity Version 2022.3.2f1 (d74737c6db50)
 
-Download and Unzip SynthesisOSX[Version Number].zip.
-Navigate to the Synthesis.pkg, and use that to install Synthesis.
+These dependencies need to be satisfied before attempting to build Synthesis with unity. You can either install these dependencies manually or use the *resolve dependencies* scripts (`scripts/win/resolve_deps.bat` & `scripts/osx/resolve_deps.sh` respectively) to install some of them for you.
 
-### Linux
+To automatically install .NET and Protobuf run the following script depending on your operating system:
+- Windows: `scripts/win/resolve_deps.bat`
+- MacOS & Linux: `scripts/osx/resolve_deps.sh`
 
-Download the Synthesis_x86_64.appimage file.
-You can run that directly or move it to your ~/Applications/ directory to have it registered as an app within ubuntu or whatever debian system you are using.
+Dependency Notes:
 
-The linux installer does not come with the Fusion360 Addin for exporting robots and fields.
+- The windows script requires admin privileges to install .NET and Protobuf to the system PATH. For this reason it is not included by default in `init.bat` unlike how `init.sh` includes `resolve_deps.sh`.
+- Unity must be installed manually through the Unity Hub app which can be downloaded from [https://unity.com/download](https://unity.com/download).
 
-**NOTE FOR NON DEBIAN USERS**: Synthesis can be installed and ran on other distros, however our linux package is tailored to Debian.
+### How To Build Synthesis Using An `init` Script And Unity
+
+Before attempting to build Synthesis, ensure you have all dependencies installed.
+
+1. Open a command prompt.
+2. Change directories to a location where you'd like to clone the Synthesis repository.
+3. Clone the Synthesis repository using `git clone https://github.com/Autodesk/synthesis --recurse-submodules`.
+4. Change directories into the root of the Synthesis repository and run the `init` script for your operating system.
+    - Windows: `init.bat`
+    - MacOS & Linux: `init.sh`
+5. Open `synthesis/engine` in Unity.
```

Comment:
Maybe remove this and reference the engine setup?

Reply:
I like having it set up this way just so that everything is in one place and you don't need to hop around. But if you feel that it would be better we can.

Reply:
Its fine, we can leave it